app/assets/stylesheets/galleries.css

```diff
@@ -1,5 +1,6 @@
 .gallery-box { cursor: pointer; width: 20px; text-align: center; } /* The + / - symbol to minimize a gallery */
 .gallery-title { height: 5px; } /* So that when a gallery is minimized there is spacing */
+.character-icon { width: 100px; height: 100px; }
```

Comment:
(`character-icon` is only used on `character#_editor` and `character#show` so this shouldn't affect elsewhere as is.)